autoendpoint/src/routers/apns/router.rs

```diff
@@ -335,6 +335,7 @@ mod tests {
     const DEVICE_TOKEN: &str = "test-token";
     const APNS_ID: &str = "deadbeef-4f5e-4403-be8f-35d0251655f5";
 
+    #[allow(clippy::type_complexity)]
```

Comment:
We probably *should* make this a discrete type, but we only do this once, and I don't think it kills readability.